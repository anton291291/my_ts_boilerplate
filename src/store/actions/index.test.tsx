import { PlayerActions } from './index';
import configureStore from 'redux-mock-store';

describe('test actions', () => {
    const mockStore = configureStore([]);
    const store = mockStore();
    beforeEach(() => {
        store.clearActions();
    });
    it('should create an action play', () => {
        const expectedAction = [
            {
                payload: true,
                type: 'PLAYER:PLAY'
            }
        ];

        store.dispatch(PlayerActions.setIsPlay());
        expect(store.getActions()).toMatchSnapshot();
    });
});
