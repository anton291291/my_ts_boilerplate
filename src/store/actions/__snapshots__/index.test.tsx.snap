// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test actions should create an action play 1`] = `
Array [
  Object {
    "payload": true,
    "type": "PLAYER:PLAY",
  },
]
`;
